test/README.md

```diff
@@ -137,6 +139,27 @@ Once finished with the tests and the disk, and to free the ram, simply unmount t
 sudo umount /mnt/tmp
 ```
 
+**OSX**
+
+To create a 4GB ramdisk named "ramdisk" on OSX at `/Volumes/ramdisk/`:
+
+```bash
+diskutil erasevolume HFS+ ramdisk $(hdiutil attach -nomount ram://8388608)
```

Comment:
I think this could "feel safer" for people to run if it were split into two commands (even though it's using a subshell)?

Reply:
You need to do this again at every reboot, so it's nice to have a single incantation in your bash history.

Reply:
OK, that makes sense too :)

Reply:
agree with Sjors

Reply:
OK, leaving this as one-liner for now

Reply:
doubt:
```
To create a 4 GiB RAM disk named "ramdisk" at `/Volumes/ramdisk/`:
```

Does it create a RAM disk named "ramdisk" at `/Volumes/`, resulting in `/Volumes/ramdisk/` or it creates a RAM disk named "ramdisk" into `/Volumes/ramdisk/`?

Reply:
@brunoerg 

Not sure I understand. The volume is named "ramdisk" and is mounted at `/Volumes/ramdisk`. The path `/Volumes/ramdisk` does not exist before the command is executed.